spec.md

```diff
@@ -100,31 +100,14 @@ The `Span` interface must have the following capabilities:
 The `Tracer` interface must have the following capabilities:
 
 - Start a `Span` that has no parent, commonly referred to as a *root* `Span` **(py: `start_trace`, go: `StartTrace`)**
-- Provide some form of access to a SpanPropagator (whether that's through embedding, inheritance, containment, or something else is per-platform)
+- Provide access to a SpanInjector and SpanExtractor that's appropriate for a specified IPC system, or a helper implementation that can be delegated to. **(py: `get_span_injector, get_span_extractor`, go: `GetSpanInjector, GetSpanExtractor`)**
```

Comment:
meta-point: like @yurishkuro, I, too, did not understand what you were getting at when I read this bullet... if we decide to proceed with this direction I would like to rewrite this to be a little clearer about the way that the injector/extractor impl is inferred.